src/buildercore/fastly/vcl/error-page.vcl.tpl

```diff
@@ -0,0 +1,5 @@
+if (obj.status == ${code}) {
```

Comment:
IIRC `vcl_error` is only for Varnish errors, so won't overwrite the backend returning 503.

Reply:
Which would be the right behavior, because nginx has custom error pages: https://github.com/elifesciences/journal-formula/blob/master/salt/journal/config/etc-nginx-sites-available-journal.conf#L146